crates/broker/src/aggregator.rs

```diff
@@ -159,21 +159,21 @@ impl AggregatorService {
         })
     }
 
-    async fn prove_assessor(&self, order_ids: &[U256]) -> Result<String> {
+    async fn prove_assessor(&self, order_ids: Vec<String>) -> Result<String> {
```

Comment:
this doesn't seem to need to clone the vec of orders? It just uses a ref to the items?
```suggestion
    async fn prove_assessor(&self, order_ids: &[String]) -> Result<String> {
```